openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_capture.json

```diff
@@ -221,6 +221,31 @@
                         {
                             "type": "splitter"
                         },
+                        {
+                            "type": "boolean",
+                            "key": "useDefaultMaterial",
+                            "label": "Use Default Material"
+                        },
```

Comment:
I feel keeping the "Display" label above this will still make it feel more like the Maya viewport's menu and will help the user navigate the settings. Thoughts?

Reply:
I agree!